sp-files/.sphinx/.markdownlint.json

```diff
@@ -0,0 +1,16 @@
+{
+    "default": false,
+    "MD003": { "style": "atx" },
+    "MD013": { "code_blocks": false, "tables": false, "stern": true, "line_length": 80},
+    "MD014": true,
```

Comment:
[We don't accept prompt marks in code samples at all](https://docs.ubuntu.com/styleguide/en/#code-examples-in-documentation).

Reply:
For the time being, would it make sense to include the linter without a config file (with just it's default settings)? Or should we move this card over to the next cycle for inclusion after all the required discussions about the rules?

Reply:
I propose we keep iterating on the config file until the end of the day. After that, let's settle on something and improve it gradually based on user feedback.

We should aim for the best default we can, but practical use will still shape it.

Reply:
Realistically the inclusion of this rule isn't a big issue - realistically it should never be triggered. I think it should be dropped purely because it doesn't align with our style guide.

I think using the rest of the config will be fine, I didn't see any issues with the other rules (other than line length). We can also expand and use more of the rules in time as well, I think some of the list formatting would be helpful.